src/api/middleware/validators/location.js

```diff
@@ -0,0 +1,13 @@
+import { query } from "express-validator";
+
+import { useExpressValidators } from "../errorHandler.js";
+import ValidationReasons from "./validationReasons.js";
+
+import LocationConstants from "../../../models/constants/Location.js";
+
+export const search = useExpressValidators([
+    query("searchTerm", ValidationReasons.DEFAULT)
+        .isString().withMessage(ValidationReasons.STRING).bail()
+        .isLength({ min: 3 }).withMessage(ValidationReasons.LOCATION_SEARCH_TERM_MIN_LENGTH(LocationConstants.searchTerm.minLength)),
```

Comment:
```suggestion
        .isLength({ min: LocationConstants.searchTerm.minLength }).withMessage(ValidationReasons.LOCATION_SEARCH_TERM_MIN_LENGTH(LocationConstants.searchTerm.minLength)),
```

Reply:
There is a `TOO_SHORT` validation reason that should be reused. THe error object will already tell you which field it is, no need to create more and more reasons.